python/ray/rllib/ddpg2/ddpg.py

```diff
@@ -0,0 +1,260 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pickle
+import os
+
+import numpy as np
+import tensorflow as tf
+
+import ray
+from ray.rllib import optimizers
+from ray.rllib.ddpg2.ddpg_evaluator import DDPGEvaluator
+from ray.rllib.agent import Agent
+from ray.tune.result import TrainingResult
+
+
+OPTIMIZER_SHARED_CONFIGS = [
+    "buffer_size", "prioritized_replay", "prioritized_replay_alpha",
+    "prioritized_replay_beta", "prioritized_replay_eps", "sample_batch_size",
+    "train_batch_size", "learning_starts", "clip_rewards"]
+
+DEFAULT_CONFIG = dict(
+    # === Model ===
+    # Hidden layer sizes of the policy networks
+    actor_hiddens=[64, 64],
+    # Hidden layer sizes of the policy networks
+    critic_hiddens=[64, 64],
+    # N-step Q learning
+    n_step=1,
+    # Config options to pass to the model constructor
+    model={},
+    # Discount factor for the MDP
+    gamma=0.99,
+    # Arguments to pass to the env creator
+    env_config={},
+
+    # === Exploration ===
+    # Max num timesteps for annealing schedules. Exploration is annealed from
+    # 1.0 to exploration_fraction over this number of timesteps scaled by
+    # exploration_fraction
+    schedule_max_timesteps=100000,
+    # Number of env steps to optimize for before returning
+    timesteps_per_iteration=1000,
+    # Fraction of entire training period over which the exploration rate is
+    # annealed
+    exploration_fraction=0.1,
+    # Final value of random action probability
+    exploration_final_eps=0.02,
+    # UO-noise scale
+    noise_scale=0.1,
+    # theta
+    exploration_theta=0.15,
```

Comment:
sorry, I am not familiar with stochastic processes and do not have such an insight on the two parameters. I will learn it and revise later. Well, I observed that most users of a RL package know OU from the DDPG paper and thus theta and sigma may make more sense instead of an intuitive and insightful name. LoL

Reply:
Yeah, I was only suggesting adding another comment, not changing the name